src/cljs/ataru/hakija/application_hakukohde_component.cljs

```diff
@@ -210,20 +212,34 @@
    [:h2 @(subscribe [:application/hakukohteet-header])]
    [scroll-to-anchor field-descriptor]])
 
+(defn- select-new-hakukohde-row []
+  (when @(subscribe [:application/hakukohteet-editable?])
+    ;[:
+    [:div
+     [:div.application__hakukohde-row--search-toggle
+      {:on-click hakukohde-search-toggle-event-handler
+       :class
+       (clojure.string/join " " [(when (not @(subscribe [:application/show-hakukohde-search]))
+                                   "application__hakukohde-row--search-toggle--closed")
+                                 (when @(subscribe [:application/prioritize-hakukohteet?])
+                                   "application__hakukohde-row--search-toggle--prioritized")])}
+      [:div.application__hakukohde-row-icon-container
+       [:i.zmdi.zmdi-graduation-cap.zmdi-hc-3x]]
+      [:a.application__hakukohde-selection-open-search
+
+       (get-translation :add-application-option)
+       (when-let [max-hakukohteet "6"]
```

Comment:
Tässä pitäis varmaankin "6" sijaan olla `@(subscribe [:application/max-hakukohteet])`